src/main/java/com/exadel/discount/entity/Discount.java

```diff
@@ -0,0 +1,59 @@
+package com.exadel.discount.entity;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.UUID;
+
+@Getter @Setter
+@EqualsAndHashCode
+@NoArgsConstructor
+@Entity
+@Table(name = "discounts")
+public class Discount {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private UUID id;
+
+    @ManyToOne(fetch = FetchType.EAGER, optional = false)
+    @JoinColumn(name = "category_id")
+    private Category category;
+
+    private String name;
+    private String description;
+    private String promo;
+    private int percent;
+    private Date start_time;
+    private Date end_time;
+    private boolean active;
```

Comment:
Maybe @Column(name = "...") annotation will be good for each field, but your variables should be startTime instead_of start_time

Reply:
+